apps/mark/frontend/src/pages/replace_election_screen.tsx

```diff
@@ -51,7 +51,9 @@ export function ReplaceElectionScreen({
         await makeCancelable(getElectionDefinitionFromCard())
       );
     })();
-  }, [getElectionDefinitionFromCard, makeCancelable]);
+    // Try to read scanner report data from the card once on mount
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
A case basically identical to the one above: A read that needs to happen just once on component mount that's also triggered by card insertion and that also infinite loops when all dependencies are listed